plugins/win-capture/window-helpers.c

```diff
@@ -413,3 +471,27 @@ HWND find_window(enum window_search_mode mode, enum window_priority priority,
 
 	return best_window;
 }
+
+HWND find_window_one_of(enum window_search_mode mode, const DARRAY(struct game_capture_picking_info) * games_whitelist)
```

Comment:
Same here, a bit confused by the name of the function. Suggestion: parse_opened_windows, would this be more descriptive?

Reply:
With functions "find_window_one_of()" and "window_rating_by_list()" I tried to keep naming similar to name used for mode with exact window selected "find_window()" and "window_rating()".